### ISSUE TITLE
ui5-label: being read as "clickable"

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#5601]
### **Bug Description**
The ui5-label is being read as "clickable {label-text}" by screenreader.  

### **Expected Behavior**
The ui5-label should not be announced as "clickable"

### **Steps to Reproduce**
1. Go to https://stackblitz.com/edit/js-emgby8 with NVDA screenreader running
2. open the dialog
3. press the UP arrow key
4. notice its reads "clickable TEST LABEL"

### **Context**
- UI5 Web Components version: 1.6.0

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by MapTo0** [comment]
Hello @SAP/ui5-webcomponents-topic-rd 

I simplified the example: https://stackblitz.com/edit/js-gm5vas?file=index.html,index.js

**The issue is not reproducible with JAWS**

Please confirm that this is a limitation of NVDA and close the issue.

Regards,
Martin

**Comment by TeodorTaushanov** [comment]
Hi,

This is up to NVDA screen reader. When there is a "click" event attached to some HTML element, NVDA reads it "clickable".

There are a lot of complaints about this:
[](https://webaim.org/discussion/mail_thread?thread=8153#:~:text=You%20can%20tell%20NVDA%20to,t%20report%20that%20information%20anymore.)

We cannot change this NVDA behavior. 

JAWS screen reader reads it correctly.

Best,
Teo
